CHANGELOG.md

```diff
@@ -52,7 +52,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Fixed
 
-- We fixed an issue where an exception occurred when a linked online file was edited in the entry editor [#8008](https://github.com/JabRef/jabref/issues/8008)
+- We fixed an issue where an exception occurred when pasting an entry with a publication date-range of the form 1910/1917 [#7864](https://github.com/JabRef/jabref/issues/7864)
```

Comment:
There seems to be some issues with your branch

Reply:
thanks very much for pointing this out. I will fix the changelog.

Reply:
Sorry about that. I think it is fixed now, please let me know if there is any more changes needed.